_includes/neurips/2020/about.neurips.html

```diff
@@ -42,16 +41,16 @@ <h2>Call for Participation</h2>
       <div class="contents">
         <h3>Important Dates</h3>
         <ul>
-            <li><strong>September 15th,  2020:</strong> <a target="_blank" href="https://neurips.cc/Register2"> Registration</a> to conference opens</li>
+            <li><strong>September 15th,  2020:</strong> <a href="https://neurips.cc/Register2"> Registration</a> to conference opens</li>
 
             <!-- <li style="text-decoration: line-through;"><strong>April 28th, 2020:</strong> [LatinX Mentoring Program]
             <a target="_blank" href=" https://docs.google.com/forms/d/e/1FAIpQLSd5Mz___3G3jVOxoTbvCPv8SIn_jTpoXzzUxTknO1gjQUuiQA/viewform?usp=sf_link
 ">Call for Mentees</a> Deadline</li> -->
             <!-- <li style="text-decoration: line-through;"><strong>May 6th, 2020:</strong> [LatinX Mentoring Program]
                 <a target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLScaw0QMM9DWMKUxak4Z96AiIUlP-9kVRfnqijtemP1p05tqCA/viewform?usp=sf_link">
                     Call for Mentors</a> Deadline</li> -->
-            <li><strong>October 7th, 2020:</strong> <a target="_blank" href="https://cmt3.research.microsoft.com/LXAINeiurIPS2020">Abstract submission</a> deadline</li>
```

Comment:
Please do not remove the target="_blank" for external links - this allows the link to open in a new tab.